src/coord/axisCommonTypes.ts

```diff
@@ -244,6 +244,10 @@ interface MinorTickOption {
 
 interface SplitLineOption {
     show?: boolean,
+    // null/undefined (true) | true | false | 'auto' (true when nick tick only)
+    showMinLine?: boolean | 'auto',
+    // null/undefined (true) | true | false | 'auto' (true when nick tick only)
```

Comment:
`null/undefined` is not necessary and they should not have a default value defined here. Instead, the default value should be defined in `axisDefault.ts`.